api/proto/core/pipeline/cron/cron.proto

```diff
@@ -47,6 +47,48 @@ service CronService {
       put: "/api/pipeline-crons/{cronID}",
     };
   }
+
+  rpc CronSave (CronSaveRequest) returns (CronSaveResponse) {
+    option (google.api.http) = {
+      post: "/api/pipeline-crons/action/save",
```

Comment:
`action` -> `actions`

what's the difference between `create` and `save`?

Reply:
In order to provide the old insertorupdatepipelinecron method for cron, I changed the name to keep consistent